strings.txt

```diff
@@ -9261,6 +9261,40 @@ SETUP_RELEASE_TYPES_DESC
 	FR	Vous pouvez spécifier quels types de sorties (voir <a href="https://musicbrainz.org/doc/Release_Group/Type">MusicBrainz</a>) doivent être inclus dans la liste des albums. Si aucun type n'a été défini pour une sortie, le type "Album" est pris par défaut. Le type "Album" ne peut pas être exclu.
 	NL	Je kunt opgeven welke releasetypes je wilt opnemen in de albumlijst (zie <a href="https://musicbrainz.org/doc/Release_Group/Type">MusicBrainz</a>). Als voor een release geen type is gedefinieerd, wordt uitgegaan van 'Album'. Albums kunnen niet van die lijst worden uitgesloten.
 
+SETUP_RELEASE_TYPES_GROUP_OPTIONS
+	EN	Release type grouping options
+	FR	Options de regroupement des types de sorties
+	DE	Gruppierungsoptionen für Release-Typen
+	NL	Groeperingsopties voor het releasetype
```

Comment:
Shouldn't this simply be "Composer Albums" or similar, as the options aren't that much about releases in general, but really specific to those items?